api/api.py

```diff
@@ -369,22 +368,22 @@ async def post_project_add(project: Project):
     if project.url is None or project.url.strip() == '':
         return ORJSONResponse({'success': False, 'err': 'URL is empty'})
 
-    project.url = escape(project.url, quote=False)
+    project.url = sanitize(project.url)
 
     if project.name is None or project.name.strip() == '':
         return ORJSONResponse({'success': False, 'err': 'Name is empty'})
 
-    project.name = escape(project.name, quote=False)
+    project.name = sanitize(project.name)
 
     if project.email is None or project.email.strip() == '':
         return ORJSONResponse({'success': False, 'err': 'E-mail is empty'})
 
-    project.email = escape(project.email, quote=False)
+    project.email = sanitize(project.email)
 
     if project.branch.strip() == '':
         project.branch = None
     else:
-        project.branch = escape(project.branch, quote=False)
+        project.branch = sanitize(project.branch)
```

Comment:
Please no call for every single variable. Just make one loop at the end that goes over all the accessible attributes of the object or even better: Extend your sanitze function to accept objects